.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+Please give a clear and concise description of what the bug is.
+
+**To Reproduce**
+If you can describe the steps needed to reproduce here that will help us squash the bug faster.  If you can reproduce the bug using simple code, please put that code here.  Github supports [markdown](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax), you can surround code snippets by three backticks \`\`\` to show the code elegantly in the bug.
```

Comment:
Since this is a template, you can put formatting elements in for the user instead o describing how to put formatting elements in:

    ```
    import cirq
    cirq.cause_bug()
    ```

Reply:
I agree - we can cut most of the description here with the code template. 
Plus, for the output, you can do: 

```
<details>
my output
</details> 
```

Which will create a foldable output for long outputs, keeping the bug tidy.